colin/checks/dockerfile.py

```diff
@@ -17,18 +17,11 @@
 from colin.core.checks.dockerfile import DockerfileAbstractCheck, InstructionCountAbstractCheck
 from colin.core.result import CheckResult
 from colin.core.target import ImageName
+from colin.core.checks.fmf_check import FMFAbstractCheck
 
 
-class FromTagNotLatestCheck(DockerfileAbstractCheck):
-    name = "from_tag_not_latest"
-
-    def __init__(self):
-        super(FromTagNotLatestCheck, self) \
-            .__init__(message="In FROM, tag has to be specified and not 'latest'.",
-                      description="Using the 'latest' tag may cause unpredictable builds."
-                                  "It is recommended that a specific tag is used in the FROM.",
-                      reference_url="https://fedoraproject.org/wiki/Container:Guidelines#FROM",
-                      tags=["from", "dockerfile", "baseimage", "latest"])
+class FromTagNotLatestCheck(FMFAbstractCheck, DockerfileAbstractCheck):
+    name, metadata = FMFAbstractCheck.get_metadata("from_tag_not_latest")
```

Comment:
I wonder whether there is a nicer way of doing this. Calling functions on class' attributes outside of methods seems sketchy.

Reply:
Yep, I had also troubles with this, but actually there is probably not better way how to do it without bigger changes in colin. Because current colin implementation does static inspection of classes, to find checks based on class attribute ``name``. If you have some idea I'll be happy to change it to better way